docs/api/graph/tick.md

```diff
@@ -1,4 +1,4 @@
-The following tick label modes are supported:
+The following tick (Y axis) label modes are supported:
```

Comment:
My main concern with this wording is that users might get confused with the overall y-axis label.